src/data/sources.json

```diff
@@ -41,8 +41,8 @@
   },
   {
     "name": "CNN",
-    "site": "https://edition.cnn.com/",
-    "base": "" 
+    "site": "https://edition.cnn.com/business",
+    "base": ""
```

Comment:
Here is where the base needs to be set. Be sure to omit the trailing '/' as the urls to be appended to it already include one